service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AisConsentCreationSteps.java

```diff
@@ -82,28 +85,15 @@ public void psu_created_a_consent_on_dedicated_account(String accounts, String b
         request.toHttpEntity(),
         ConsentsResponse201.class);
 
-    context.setConsentId(response.getBody().getConsentId());
-  }
-
-  private List<AccountReference> fillAccountReferences(String[] ibans) {
-    ArrayList<AccountReference> result = new ArrayList<>();
-
-    if (ibans[0].equals("null")) {
-      return null;
+    if (scaApproach.toLowerCase().equals("redirect")) {
```

Comment:
There is also a `equalsIgnoreCase` method (occurs a few times in this change)

Reply:
Done